### PR TITLE
Add option to collapse whitespace containing newlines to `\n` instead of ` `

### PR DESCRIPTION
**Pull request by stuff**
I need to convert custom markdown to html and convert back this html to original markdown, and carriage returns are useful to me. I made this small change and think it may be useful for someone else :)

### CODE REVIEW

**Comment by wooorm** [comment]
Alright! Do you have an example of a higher level textual input/output markdown/html results, and how this options changes it?

File: index.js [hunk 1]
```diff
@@ -2,13 +2,16 @@
...
-var minify = require('rehype-minify-whitespace')();
+var rehypeMinifyWhitespace = require('rehype-minify-whitespace');
 var xtend = require('xtend');
 var one = require('./lib/one');
 var handlers = require('./lib/handlers');
 
 function toMDAST(tree, options) {
   var settings = options || {};
+  var minify = rehypeMinifyWhitespace({
+    newlines: typeof settings.newlines === 'boolean' ? settings.newlines : false
```
Comment (wooorm) [inline index.js]: Could we just do `settings.newlines === true` here? That does the same right, and is a bit more readable?

**Comment by stuff** [comment]
Well. After some more thinking, those newlines have no effect on rendering, and I found a better way to fix my problem without modifying hast-util-to-mdast. Unless someone think it's a good change, I think we can close the PR and forget it completely

**Comment by wooorm** [comment]
haha, okay, well if it isn’t needed... 🤷‍♂️
Anyway, thanks for working on it and helping though!

**Pull request closed without merging**